index.js

```diff
@@ -330,6 +342,8 @@ Polyglot.prototype.has = function (key) {
 };
 
 // export transformPhrase
-Polyglot.transformPhrase = transformPhrase;
+Polyglot.transformPhrase = function (phrase, substitutions, locale) {
```

Comment:
let's name this (`function transformPhrase`)

Reply:
sure